### DISCUSSION TITLE
Faceted track selector and reusable track sets

### DISCUSSION THREAD

**Discussion by whtns** [GMOD/jbrowse-components#1706] [General] [answered]
I use jbrowse to view pileups of single cells. I used a [faceted track selector](https://jbrowse.org/docs/track_selectors.html) and added [metadata](https://jbrowse.org/docs/track_metadata.html) with jbrowse 1 to view subsets of single cell experiments. I was also able to access track sets for individual experiments at different urls (i.e. <hostname/jbrowse/?data=mydata>) are either of these functions available in jbrowse 2?

**Comment by cmdcolin** [comment, marked as answer]
@whtns we don't have a faceted track selector, but if coarse grained categories help, you can load your tracks with `--category MyCategory` or comma separated `--category Nested,SubCategory`

with the individual experiments, you can create multiple config files, and then navigate to them like this

http://s3.amazonaws.com/jbrowse.org/code/jb2/master/index.html?config=test_data/config_demo.json

You can use the --out parameter to create custom output paths with the command line tool, so on the command line we could run

    npm install -g @jbrowse/cli
    mkdir hg19
    jbrowse add-assembly hg19.fa --out hg19 --load copy
    jbrowse add-track myreads.bam --out hg19 --copy copy

This will output to a directory named hg19 with a config file hg19/config.json and you could move this directory into /var/www/html/jbrowse2/hg19 or similar and access http://localhost/jbrowse2/?config=hg19/config.json

Hope that helps :)

We have https://github.com/GMOD/jbrowse-components/issues/1707 for tracking the creation of the faceted track selector
